Gaijinizer patch/CommonEvents_428.txt

```diff
@@ -4,80 +4,96 @@
 
 絶対に離さなイヨ…♡
 > CONTEXT: CommonEvents_428/14/Dialogue/enemy13
-No...♡. Iyo absolutely Iyo...♡#MTLed
+nO...♡
+
+I wiLL abSOluteLY not leT gO...♡#Strate
 > END STRING
 
 > BEGIN STRING
 いやよ…！！
 
 絶対に離さないから…！
 > CONTEXT: CommonEvents_428/20/Dialogue/enemy12
-no…! ! From I won't let go ...!#MTLed
+No...!!
+
+I absolutely won't let go...!#Strate
 > END STRING
 
 > BEGIN STRING
 ………離シて欲しイ……の？
 > CONTEXT: CommonEvents_428/30/Dialogue/enemy13
-......... I want to leave me .........#MTLed
+.........yoU want mE to......leAvE?#Strate
 > END STRING
 
 > BEGIN STRING
 離して欲しいの……？
 
 …………
 > CONTEXT: CommonEvents_428/34/Dialogue/enemy12
-Do you want me to leave ...? ............#MTLed
+You want me to leave......?
+
+............#Strate
 > END STRING
 
 > BEGIN STRING
 …………今回だケ
 
 特別……ダよ…♡
 > CONTEXT: CommonEvents_428/44/Dialogue/enemy13
-............ This time. Special ...... Da......
-♡#MTLed
+............onLY thiS OncE.
+
+I'll mAke......aN exCepTion......♡#Strate
 > END STRING
 
 > BEGIN STRING
 ダメ……♡
 
 \\N[1]は私ト、ずっト一緒にいルの♡
 > CONTEXT: CommonEvents_428/58/Dialogue/enemy13
-No...♡. \\N[1] is my friend, I am together#MTLed
+nO...♡
+
+I'll bE toGEther with \\N[1], forEveR♡#Strate
```

Comment:
If you see \\N[1] or someone else in conversation being referred to by their name instead of a pronoun, feel free to translate it to "you" where it would sound more natural instead of following the Japanese convention of using people's names all the time.